app/src/lib/stats/stats-store.ts

```diff
@@ -257,6 +259,8 @@ type DailyStats = ICalculatedStats &
 export interface IStatsStore {
   recordMergeAbortedAfterConflicts: () => void
   recordMergeSuccessAfterConflicts: () => void
+  recordRebaseAbortedAfterConflicts: () => void
```

Comment:
**Non-blocking question**
Is this intended to make testing easier?

Reply:
This is because we use `IStatsStore` rather than `StatsStore` directly in `changes-state.ts`.

The benefit of this interface is that we can write tests to verify these side effects are invoked in the right scenarios.